src/components/Save.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class Save extends Component {
+	render() {
+		return (
+			<div className="save">
+				<i onClick={ this.props.saveChanges } className="far fa-save fa-lg icon"></i>
```

Comment:
Since you only convey meaning by an icon, you should add an `aria-label` attribute (it stays `aria-label`, even in JSX) to name the action.
ex. : <i aria-label='Save'>...